NetCasbin/Rbac/IRoleManager.cs

```diff
@@ -7,47 +7,43 @@ public interface IRoleManager
     {
         /// <summary>
         /// Clear clears all stored data and resets the role manager to the initial state.
-        /// 
         /// 清除所有的数据并将角色管理器重置为初始状态
         /// </summary>
         void Clear();
 
         /// <summary>
-        /// addLink adds the inheritance link between two roles. role: name1 and role:
+        /// Adds the inheritance link between two roles. role: name1 and role:
         /// name2. domain is a prefix to the roles.
-        /// 
         /// 新增两个角色之间的继承关系
```

Comment:
You may want to translate these chinese comments into English :smile:

Reply:
I found the English version is above at it..., It looks the Chinese version was translated from that.

Reply:
Remove all Chinese.

Reply:
> Remove all Chinese.

Ok.